linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/CoreJSLib.scala

```diff
@@ -413,12 +415,13 @@ private[emitter] object CoreJSLib {
       val toStr = {
         MethodDef(static = false, Ident("toString"), Nil, {
           Return(Apply(genIdentBracketSelect(StringRef, "fromCharCode"),
-              (This() DOT "c") :: Nil))
+                  (This() DOT "c") :: Nil))
         })
       }
 
       if (useECMAScript2015) {
-        buf += extractWithGlobals(globalClassDef("Char", CoreVar, None, ctor :: toStr :: Nil))
+        buf += extractWithGlobals(globalClassDef("Char", CoreVar, None,
+                ctor :: toStr :: Nil))
```

Comment:
This indentation seems plain incorrect. Indenting by 4 spaces *per brace* (which I assume is happening here) seems insane :S

Reply:
That is what seems to be happening. I think this might be a bug in scalafmt, so I'll open an issue for it. But this particular case was also fixed by increasing the line length to 100.

Reply:
It turns out that there already is an issue for this: scalameta/scalafmt#2079

Reply:
Is dropping binpacking up for discussion at this point? Personally, I'd rather go with a well supported formatting style than sticking to a particular other style. Especially considering that breaking blame seems to become less of a concern. @sjrd?

Reply:
I have no faith in the non-binpacking formatting, especially not for our codebase. We have many methods with many arguments, but where each argument is typically short. This goes nicely with binpacking, since most method calls can stay within 2 lines. Without binpacking, we'll waste a lot of vertical space.